kod/object/active/holder/nomoveon/battler/monster.kod

```diff
@@ -5180,6 +5192,14 @@ messages:
       {
          Send(poOwner,@SomeoneSaid,#type=SAY_RESOURCE,#what=self,
               #string=monster_teach_impossible);
+         
+         if Send(who,@GetBaseMaxHealth) = 100+Send(who,@GetStamina)
```

Comment:
This should probably be if Send(who,@GetBaseMaxHealth) >= 100 + Send(who,@GetRawStamina). There are cases where a player may have more than 100 + stamina HP, such as someone with 40 stam and a duke shield with 145HP who then loses his +5 bonus (so your equation would yield if 145 = 143 and return FALSE). They could of course die to remove these HP or boost their shield, but I guarantee people will instead complain that the system is broken.